Crash on startup in Android 9

## Exception
* User Action: UI Error
* Request: Application crash
* Version: 3.5.2
* OS: Linux Android 10 - 29
* Device: HWCLT
* Model: CLT-L29
* Product: CLT-L29
<details><summary><b>Crash log </b></summary><p>

```java.lang.RuntimeException: Unable to start activity ComponentInfo{com.amaze.filemanager/com.amaze.filemanager.ui.activities.MainActivity}: android.view.InflateException: Binary XML file line #78 in com.amaze.filemanager:layout/main_toolbar: Binary XML file line #78: You must supply a layout_width attribute.
 at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3782)
 at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3961)
 at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:91)
 at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:149)
 at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:103)
 at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2386)
 at android.os.Handler.dispatchMessage(Handler.java:107)
 at android.os.Looper.loop(Looper.java:213)
 at android.app.ActivityThread.main(ActivityThread.java:8178)
 at java.lang.reflect.Method.invoke(Native Method)
 at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:513)
 at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1101)
Caused by: android.view.InflateException: Binary XML file line #78 in com.amaze.filemanager:layout/main_toolbar: Binary XML file line #78: You must supply a layout_width attribute.
Caused by: java.lang.UnsupportedOperationException: Binary XML file line #78: You must supply a layout_width attribute.
 at android.content.res.TypedArray.getLayoutDimension(TypedArray.java:990)
 at android.view.ViewGroup$LayoutParams.setBaseAttributes(ViewGroup.java:8032)
 at android.view.ViewGroup$MarginLayoutParams.<init>(ViewGroup.java:8230)
 at androidx.drawerlayout.widget.DrawerLayout$LayoutParams.<init>(DrawerLayout.java:2319)
 at androidx.drawerlayout.widget.DrawerLayout.generateLayoutParams(DrawerLayout.java:1906)
 at android.view.LayoutInflater.rInflate(LayoutInflater.java:1135)
 at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:1094)
 at android.view.LayoutInflater.inflate(LayoutInflater.java:692)
 at android.view.LayoutInflater.inflate(LayoutInflater.java:536)
 at android.view.LayoutInflater.inflate(LayoutInflater.java:483)
 at androidx.appcompat.app.AppCompatDelegateImpl.setContentView(AppCompatDelegateImpl.java:555)
 at androidx.appcompat.app.AppCompatActivity.setContentView(AppCompatActivity.java:161)
 at com.amaze.filemanager.ui.activities.MainActivity.onCreate(MainActivity.java:273)
 at android.app.Activity.performCreate(Activity.java:8086)
 at android.app.Activity.performCreate(Activity.java:8074)
 at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1313)
 at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3755)
 at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3961)
 at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:91)
 at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:149)
 at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:103)
 at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2386)
 at android.os.Handler.dispatchMessage(Handler.java:107)
 at android.os.Looper.loop(Looper.java:213)
 at android.app.ActivityThread.main(ActivityThread.java:8178)
 at java.lang.reflect.Method.invoke(Native Method)
 at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:513)
 at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1101)```

```
java.lang.RuntimeException: Unable to start activity ComponentInfo{com.amaze.filemanager/com.amaze.filemanager.ui.activities.MainActivity}: android.view.InflateException: Binary XML file line #107 in com.amaze.filemanager:layout/main_toolbar: Binary XML file line #107: You must supply a layout_width attribute.
at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3539)
at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3699)
at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:85)
at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135)
at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2135)
at android.os.Handler.dispatchMessage(Handler.java:106)
at android.os.Looper.loop(Looper.java:236)
at android.app.ActivityThread.main(ActivityThread.java:8037)
at java.lang.reflect.Method.invoke(Native Method)
at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:656)
at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:967)
Caused by: android.view.InflateException: Binary XML file line #107 in com.amaze.filemanager:layout/main_toolbar: Binary XML file line #107: You must supply a layout_width attribute.
Caused by: java.lang.UnsupportedOperationException: Binary XML file line #107: You must supply a layout_width attribute.
at android.content.res.TypedArray.getLayoutDimension(TypedArray.java:832)
at android.view.ViewGroup$LayoutParams.setBaseAttributes(ViewGroup.java:8322)
at android.view.ViewGroup$MarginLayoutParams.(ViewGroup.java:8520)
at androidx.drawerlayout.widget.DrawerLayout$LayoutParams.(DrawerLayout.java:2319)
at androidx.drawerlayout.widget.DrawerLayout.generateLayoutParams(DrawerLayout.java:1906)
at android.view.LayoutInflater.rInflate(LayoutInflater.java:1127)
at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:1086)
at android.view.LayoutInflater.inflate(LayoutInflater.java:684)
at android.view.LayoutInflater.inflate(LayoutInflater.java:536)
at android.view.LayoutInflater.inflate(LayoutInflater.java:479)
at androidx.appcompat.app.AppCompatDelegateImpl.setContentView(AppCompatDelegateImpl.java:696)
at androidx.appcompat.app.AppCompatActivity.setContentView(AppCompatActivity.java:170)
at com.amaze.filemanager.ui.activities.MainActivity.onCreate(MainActivity.java:326)
at android.app.Activity.performCreate(Activity.java:8157)
at android.app.Activity.performCreate(Activity.java:8129)
at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1310)
at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3512)
at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3699)
at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:85)
at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135)
at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2135)
at android.os.Handler.dispatchMessage(Handler.java:106)
at android.os.Looper.loop(Looper.java:236)
at android.app.ActivityThread.main(ActivityThread.java:8037)
at java.lang.reflect.Method.invoke(Native Method)
at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:656)
at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:967)
```

--- a/app/src/main/res/layout-w720dp/layout_appbar.xml
+++ b/app/src/main/res/layout-w720dp/layout_appbar.xml
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <com.google.android.material.appbar.AppBarLayout
     xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
     android:elevation="4dp"
@@ -9,8 +10,6 @@
     >
 
     <androidx.appcompat.widget.Toolbar
-        xmlns:app="http://schemas.android.com/apk/res-auto"
-        xmlns:android="http://schemas.android.com/apk/res/android"
         android:id="@+id/action_bar"
         android:layout_height="?attr/actionBarSize"
         android:layout_width="match_parent"

